### ISSUE TITLE
HI Team, I am using webview_flutter plug for loading sub widget on my screen. While entering into WebView with Accessibility the screen navigation lost the other widgets like back button and app bar navigation. This issue happens only by WebView scrollable content. After entering into WebView scroll then the screen navigation lost their other widget. Accessi

### ISSUE THREAD

**Issue by tamminivenkat** [flutter/flutter#87030]
## Steps to Reproduce

1. Run `flutter create bug`.
2. Update the files as follows: ... 
3. ... 

**Expected results:** 

**Actual results:** 

<details>
  <summary>Logs</summary>

```
```

```
```

```
```

</details>

**Comment by darshankawar** [comment]
@tamminivenkat 
Can you provide `flutter doctor -v` and a [complete minimal reproducible code sample](https://stackoverflow.com/help/minimal-reproducible-example) that shows this issue ?
Thanks.

**Comment by tamminivenkat** [comment]
Here is the sample code which blocking the accessibility navigation(top to bottom navigation with left and right swiping).  Here I am unable to get appBar items after reading WebView reading content with scrolling some. and also favorite button I didn't get navigation. Within Webview, I can able to left and right swiping only. And #87031 is different issue.

**Comment by tamminivenkat** [comment]
Here, I did test WebView on your WebView plugin sample. I'm getting same issue on this code too.

https://github.com/flutter/plugins/tree/master/packages/webview_flutter/webview_flutter/example

**Comment by darshankawar** [comment]
@tamminivenkat 
Check [this](https://github.com/flutter/flutter/issues/30804) open issue and see if it resembles / is similar to your case.

**Comment by tamminivenkat** [comment]
Not able to come out of webview using swipe navigation in voice over mode.

**Comment by darshankawar** [comment]
Maybe similar to https://github.com/flutter/flutter/issues/30804

I see the same behavior using webview plugin [example](https://github.com/flutter/plugins/tree/master/packages/webview_flutter/webview_flutter/example). Verified on iPhone 6s (OS 14.4.1) on latest stable.

<details>
<summary> flutter doctor -v </summary>

```
[✓] Flutter (Channel stable, 2.2.3, on Mac OS X 10.15.4 19E2269 darwin-x64,
    locale en-GB)
    • Flutter version 2.2.3 at /Users/dhs/documents/fluttersdk/flutter
    • Framework revision f4abaa0735 (5 weeks ago), 2021-07-01 12:46:11 -0700
    • Engine revision 241c87ad80
    • Dart version 2.13.4

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.3, Build version 12C33
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] VS Code (version 1.58.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.21.0

[✓] Connected device (4 available)
    • Darshan's iphone (mobile)  • 21150b119064aecc249dfcfe05e259197461ce23 •
      ios            • iOS 14.4.1
    • iPhone 12 Pro Max (mobile) • A5473606-0213-4FD8-BA16-553433949729     •
      ios            • com.apple.CoreSimulator.SimRuntime.iOS-14-3 (simulator)
    • macOS (desktop)            • macos                                    •
      darwin-x64     • Mac OS X 10.15.4 19E2269 darwin-x64
    • Chrome (web)               • chrome                                   •
      web-javascript • Google Chrome 92.0.4515.107

• No issues found!



```

</details>
